src/runtime/database.cpp

```diff
@@ -1070,6 +1090,44 @@ static std::vector<JobReflection> find_all(Database *db, sqlite3_stmt *query, bo
   return out;
 }
 
+static std::vector<JobReflection> get_all_jobs_visualization(Database *db, sqlite3_stmt *query) {
+    const char *why = "Could not get job visualization";
+    std::vector<JobReflection> out;
+
+    db->begin_txn();
+    while (sqlite3_step(query) == SQLITE_ROW) {
+        JobReflection reflection = find_one(db, query, true);
+        reflection.starttime = std::to_string(sqlite3_column_int64(query, 7));
+        reflection.endtime = std::to_string(sqlite3_column_int64(query, 8));
```

Comment:
Maybe we should introduce a new 'Time' type for these fields and it supports .asInt64() and .asString() ?

Reply:
Done